src/main/kotlin/com/example/mylotto/service/LottoService.kt

```diff
@@ -3,24 +3,32 @@ package com.example.mylotto.service
 import com.example.mylotto.constant.LottoConstant
 import com.example.mylotto.enum.Rank
 import com.example.mylotto.model.LottoTicket
+import com.example.mylotto.model.LottoTicketOrder
 import com.example.mylotto.model.LottoWinningNumbers
 
 class LottoService {
-    fun generateLottoTickets(purchaseAmount: Long): List<LottoTicket> {
+    fun calculateCount(purchaseAmount: Long): Int {
         require(purchaseAmount > 0 && purchaseAmount % LottoConstant.LOTTO_TICKET_PRICE == 0L) {
             "Purchase amount must be a positive multiple of ${LottoConstant.LOTTO_TICKET_PRICE}."
         }
-        val ticketCount = (purchaseAmount / LottoConstant.LOTTO_TICKET_PRICE).toInt()
-        return List(ticketCount) {
-            LottoTicket()
-        }
+        return (purchaseAmount / LottoConstant.LOTTO_TICKET_PRICE).toInt()
+    }
+
+    fun generateLottoTickets(order: LottoTicketOrder): List<LottoTicket> {
+        val manualTickets = order.manualTickets
+        val automaticTickets =
+            (1..order.automaticCount).map {
+                LottoTicket.ofAutomatic()
+            }
```

Comment:
```suggestion
        val automaticTickets = List(order.automaticCount) { LottoTicket.ofAutomatic() }
```
1~automaticCount만큼 for 문을 도는 것보다 이게 더 의도가 명확한 것 같은데 어떻게 생각하시나용?!

Reply:
오우.. init이 있었군요 적용해두겠습니다